modules/bot-improvement/src/backend/api.ts

```diff
@@ -17,42 +17,55 @@ interface SessionResponse extends Response {
 }
 
 export default async (bp: typeof sdk, db: Database) => {
+  const asyncMiddleware = asyncMw(bp.logger)
```

Comment:
while we're here, I just want to make sure, is this module marked as experimental ?

Reply:
sure, done!